lib/core/src/animation/physics/inertia.rs

```diff
@@ -263,6 +263,40 @@ impl PhysicsProperties {
     }
 }
 
+impl Into<PhysicsProperties> for &PhysicsProperties {
+    fn into(self) -> PhysicsProperties {
+        self.clone()
+    }
+}
+
+
+
+// ============================
+// === SimulationThresholds ===
+// ============================
+
+/// A struct holding simulation thresholds used for computing optimizations.
+#[derive(Clone,Copy,Debug)]
+pub struct SimulationThresholds {
+    /// Used to snap object to fixed point if its distance is less than this threshold.
+    pub fixed_point_distance : f32,
+    /// Used to stop simulation computing if the object speed is less than this threshold.
+    pub speed                : f32
```

Comment:
Very, Very minor thing - but lets just do it this way :) In case things are separated by docs, dont align them, it looks strange :)